Makefile

```diff
@@ -23,6 +23,22 @@ help: ## display help for this makefile
 rm-pycache: ## remove all __pycache__ files (run if encountering issues with pycharm debugger (containers exiting prematurely))
 	find . -name '__pycache__' | xargs rm -rf
 
+### Connecting to remote dbs #########################################
+remotedb: # Get a psql console on a remote db (from OSX only!)
+	export ENV=$${ENV:=rdev}; \
+	export config=$$(aws --profile $(AWS_DEV_PROFILE) secretsmanager get-secret-value --secret-id $${ENV}/aspen-config | jq -r .SecretString ); \
+	export DB_URI=$$(jq -r '"postgresql://\(.DB.admin_username):\(.DB.admin_password)@127.0.0.1:5556/$(STACK)"' <<< $$config); \
+	ssh -f -o ExitOnForwardFailure=yes -L 5556:$$(jq -r .DB.address <<< $$config):5432 $$(jq -r .bastion_host <<< $$config) sleep 10; \
```

Comment:
Open an SSH tunnel via our bastion host to the aurora DB in the background. This tunnel will CLOSE in 10 seconds if nothing is using it. Bind the aurora db's postgres port (5432) to the localhost port 5556. We're using 5556 to avoid conflicting with any other local containers or postgres services.

Reply:
10 seconds is pretty short, can we do a minute?

Reply:
The next command in these make targets (docker-compose run or psql) opens a connection to the db within our 10 second window, which prevents the connection from closing!